pkg/base1/test-dbus-address.js

```diff
@@ -3,6 +3,11 @@ import { common_dbus_tests, dbus_track_tests } from "./test-dbus-common.js";
 
 import QUnit from "qunit-tests";
 
+if (await QUnit.mock_info("bridge") == "cockpit-bridge.pyz")
+    QUnit.test.butNotForPy = QUnit.test.skip;
```

Comment:
Cool trick! Could you move this to pkg/lib/qunit-tests.js instead of duplicating it here and in test-dbus.js? We can then also use that in pkg/base1/test-http.js, two of the tests are also skipped on the pybridge.

I'd call it `.skipPybridge` to be a little more consistent with `.skip` and "pybridge", but bikeshedding :grin:

Reply:
Done!